shapers/harfbuzz.lua

```diff
@@ -42,16 +42,23 @@ function shaper:shapeToken (text, options)
     SU.warn("Font family '"..options.family.."' not available, falling back to '"..face.family.."'")
   end
   usedfonts[face] = true
+  local point_size = ("%g"):format(SILE.measurement(options.size):tonumber())
+  -- If opsz axis is not explicitly set, set it to the point size.
+  local variations = options.variations
+  if not variations:find("opsz=") then
```

Comment:
Isn't the syntax `opsz = ` valid?

Reply:
I suppose, and it seems `hb_variation_from_string()` accepts also `opsz 100` without equal sign. Too bad SILE keeps these as free form  strings instead of parsing them into tables.

Reply:
Anyway, I did some refactoring locally and this is not needed anymore.